conf/sys.config

```diff
@@ -1,19 +1,22 @@
 [
-  {reconnections, []},
-  {pgsql, [
-            {host, "127.0.0.1"},
-            {username, "postgres"},
-            {password, "example"},
-            {database, "test_db"},
-            {timeout , 4000}
-          ]
-  },
-  {redis, [
-            {host , "127.0.0.1"},
-            {port , 6379},
-            {password , ""},
-            {database , 0},
-            {reconnect_sleep , 100}
-          ]
-  }
+  {reconnections, [
```

Comment:
not something to right now, but at some point these configs should only be applied to the testing and dev profiles, since they shouldn't affect apps that use this library